graphqlapi/src/test/java/io/stargate/graphql/schema/cqlfirst/dml/DmlSchemaOptionsTest.java

```diff
@@ -0,0 +1,45 @@
+package io.stargate.graphql.schema.cqlfirst.dml;
+
+import graphql.schema.GraphQLEnumValueDefinition;
+import java.util.List;
+import java.util.stream.Stream;
+import org.assertj.core.util.Lists;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class DmlSchemaOptionsTest {
+  @ParameterizedTest
+  @MethodSource
+  public void testConsistencyLevelParsing(String csvLevels, List<String> expectedValues) {
+    List<GraphQLEnumValueDefinition> values =
+        DmlSchemaBuilder.getConsistencyEnum(csvLevels, "Name").getValues();
+    // Are all values expected?
+    for (GraphQLEnumValueDefinition v : values) {
+      Assertions.assertTrue(expectedValues.contains(v.getName()));
```

Comment:
nit: for the sake of uniformity, could you use AssertJ assertions (`import static org.assertj.core.api.Assertions.assertThat`)?

That way lines 20 and 23 can be simplified as:
```
assertThat(expectedValues).contains(v.getName());
assertThat(values).hasSameSizeAs(expectedValues);
```
Or even better, replace lines 18-23 by:
```
assertThat(values)
    .extracting(GraphQLEnumValueDefinition::getName)
    .hasSameElementsAs(expectedValues);
```